README.md

```diff
@@ -0,0 +1,4 @@
+# Binance Client For Binance API
+
+source api docs: https://binance-docs.github.io
```

Comment:
pls replace this link to https://binance-docs.github.io/apidocs, current link lead to 404 page